sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -132,7 +132,6 @@ statement
 hiveNativeCommands
```

Comment:
Since we don't have `visitExecuteNativeCommand` now, do we still need this rule?